### PR TITLE
Not actual parameters

### PR DESCRIPTION
**Pull request by Golodnyi**

### CODE REVIEW

File: src/Composer/Repository/CompositeRepository.php [hunk 1]
```diff
@@ -102,7 +102,7 @@ public function search($query, $mode = 0, $type = null)
         $matches = array();
         foreach ($this->repositories as $repository) {
             /* @var $repository RepositoryInterface */
-            $matches[] = $repository->search($query, $mode, $type);
```
Comment (alcohol) [inline src/Composer/Repository/CompositeRepository.php]: While not specified in the `RepositoryInterface`, the `ComposerRepository` implementation does accept a type argument. So I would exclude this change from the PR.

Comment (Golodnyi) [inline src/Composer/Repository/CompositeRepository.php]: It is true, I made changes in my fork

**Comment by Golodnyi** [comment]
You are satisfied with the PR or create a new, without one a change?

**Comment by alcohol** [comment]
Aren't you forgetting something..? :)

**Comment by Golodnyi** [comment]
I do not know...

**Comment by Seldaek** [comment]
Thanks

**Comment by alcohol** [comment]
@Seldaek But the tests were failing.. :-/

**Comment by Golodnyi** [comment]
Time: 29.86 seconds, Memory: 57.00MB
OK (1051 tests, 2116 assertions)

¯\_(ツ)_/¯

**Comment by alcohol** [comment]
It will not run the tests on your machine if you do not have perforce installed (they will be marked as skipped). Check the travis build results. Anyway, I think @Seldaek fixed them manually on master branch already.

**Pull request merged**